docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -19,4 +19,18 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`MSCK REPAIR TABLE` recovers all the partitions in the directory of a table and updates the catalog. `MSCK REPAIR TABLE` on a non-existent table or a table without partitions throws Exception. Another way to recover partitions is to use `ALTER TABLE RECOVER PARTITIONS`.
```

Comment:
throws Exception -> throws an exception.

Is there anything else we can say about when this would become necessary? when do you need to recover a table, and is there any more to say about what it updates in the catalog / metastore? No need to say a lot, just trying to make this slightly more useful to a reader.